exercises/concept/lucas-numbers/.docs/instructions.md

```diff
@@ -0,0 +1,38 @@
+# Instructions
+
+You are a huge fan of the [Numberphile Youtube channel](https://www.youtube.com/watch?v=PeUbRXnbmms) and you just saw a cool video about the _Lucas Number Sequence_. You want to create this sequence using elixir.
+
+While designing your function, you want to make use of _lazy evaluation_, so that you can generate as many numbers as you want, but only if you need to -- So you decide to use a stream:
+
+## 1. Generate the base cases
+
+You know that the sequence has two starting numbers which don't follow the same rule. Write two base case clauses to return these numbers:
+
+```elixir
+LucasNumbers.generate(1)
+# => [2]
+
+LucasNumbers.generate(2)
+# => [2, 1]
+```
+
+## 2. Create the streaming cases
+
+For any sequence longer than 2, you know that you need to add the previous two numbers to get the next number and so on.
```

Comment:
A tiny nitpick to the wording used - there is no call to action, like "implement" or "extend".